src/fragment-components/a-progress-indicator.tsx

```diff
@@ -0,0 +1,205 @@
+	</>;
+};
+
+export const AProgressIndicatorCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <TextInput
+		value={selectedComponent.codeContext?.name}
+		labelText='Input name'
+		onChange={(event: any) => {
+			setComponent({
+				...selectedComponent,
+				codeContext: {
+					...selectedComponent.codeContext,
+					name: event.currentTarget.value
+				}
+			});
+		}}
+	/>;
+};
+
+export const AProgressIndicator = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		headingCss={css`display: block;`}
+		className={css`position: relative; display: flex`}
+		{...rest}>
+			<ProgressIndicator
+			currentIndex={componentObj.currentIndex}
+			vertical={componentObj.isVertical}
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}>
+				{
+					componentObj.progressSteps.map((step: any, index: number) => (
+						<ProgressStep {...step} key={index} />
+					))
+				}
+			</ProgressIndicator>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AProgressIndicator,
+	styleUI: AProgressIndicatorStyleUI,
+	codeUI: AProgressIndicatorCodeUI,
+	type: 'progress-indicator',
+	keywords: ['progress', 'indicator'],
+	name: 'Progress indicator',
+	defaultComponentObj: {
+		type: 'progressIndicator',
+		isVertical: true,
+		currentIndex: 0,
+		spacing: true,
+		progressSteps: [
+			{
+				label: 'Step 0',
+				description: 'Step 0: Getting started with carbon design system',
+				secondaryLabel: 'Optional Label',
+				invalid: false,
+				disabled: false
+			}
+		]
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Steps = ${json.progressSteps};
+			@Input() ${nameStringToVariableString(json.codeContext?.name)}Vertical = ${json.isVertical || false}
+			@Input() ${nameStringToVariableString(json.codeContext?.name)}Spacing = ${json.spacing || false}
+			@Input() ${nameStringToVariableString(json.codeContext?.name)}Current = ${json.currentIndex}`,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}StepSelected = new EventEmitter<Event>();`,
+			imports: ['ProgressIndicatorModule'],
+			code: ({ json }) => {
+				return `<ibm-progress-indicator
+					[steps]="${nameStringToVariableString(json.codeContext?.name)}Steps"
+					[current]="${nameStringToVariableString(json.codeContext?.name)}Current"
+					(stepSelected)="${nameStringToVariableString(json.codeContext?.name)}StepSelected.emit($event)"
+					[spacing]="${nameStringToVariableString(json.codeContext?.name)}Spacing"
+					${angularClassNamesFromComponentObj(json)}>
+				</ibm-progress-indicator>`;
+			}
+		},
+		react: {
+			imports: ['ProgressIndicator', 'ProgressStep'],
+			code: ({ json }) => {
+				return `<ProgressIndicator
+					currentIndex={state["${json.codeContext?.name}"] | 0}
```

Comment:
Should this be
```suggestion
					currentIndex={state["${json.codeContext?.name}Step"] | 0}
```
(with the onChange change as well, ofc)